### PR TITLE
Camera Ready version [Re]iFlow

### PR DESCRIPTION
**Pull request by HiddeLekanne**
Dear all,

We got it for the most part, but we still have some bugs:

- A random W on the title page overlapping with other text.
- We have to ```rm metadata.tex && make``` then use ```biber article```  and then again ```rm metadata.tex && make```to get our references working.
- We couldn't find a DOI number for the article we reproduced 

Best regards!

### CODE REVIEW

File: rijsdijk2021reproducing/openreview/packages.tex [hunk 1]
```diff
@@ -0,0 +1,26 @@
+\usepackage[final]
```
Comment (koustuvsinha) [inline rijsdijk2021reproducing/openreview/packages.tex:1]: What does this import do? Most probably the reason for the issue for compilation. Please remove this and re-try using `make` to compile

File: rijsdijk2021reproducing/openreview/packages.tex [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+\usepackage{graphicx}
+
+\usepackage{subcaption}
+\captionsetup{compatibility=false}
+
+\usepackage[normalem]{ulem}
+\newif{\ifhidecomments}
+\usepackage{notes}
+
+\usepackage[
```
Comment (koustuvsinha) [inline rijsdijk2021reproducing/openreview/packages.tex:19]: Any reason why you need to add a bibliography package like this?

File: rijsdijk2021reproducing/openreview/packages.tex (hunk 1 above)
Comment (HiddeLekanne) [reply rijsdijk2021reproducing/openreview/packages.tex]: That was under instructions of the openreview template. Now it is removed, and everything still works (better?).

File: rijsdijk2021reproducing/openreview/packages.tex (hunk 2 above)
Comment (HiddeLekanne) [reply rijsdijk2021reproducing/openreview/packages.tex]: apparently not.

**Comment by koustuvsinha** [comment]
Few more compilation errors:

```
! Missing $ inserted.
<inserted text>
                $
l.321 ...enge_2020/IMG/scores/Mcc Score_fixed.png}


LaTeX Warning: File `Reproducibility_Challenge_2020/IMG/scores/Mcc Score_fixed.
png' not found on input line 321.

! Missing $ inserted.
<inserted text>
                $
l.321 ...enge_2020/IMG/scores/Mcc Score_fixed.png}


Overfull \hbox (128.39091pt too wide) in paragraph at lines 321--322
 []\TU/SourceSerifPro(0)/m/n/10 Score$[]\OML/cmm/m/it/10 ixed:pngScore[]ixed:bb
[]$
! Missing $ inserted.
<inserted text>
                $
l.327 ...2020/IMG/scores/Log Likelihood_fixed.png}


LaTeX Warning: File `Reproducibility_Challenge_2020/IMG/scores/Log Likelihood_f
ixed.png' not found on input line 327.

! Missing $ inserted.
<inserted text>
                $
l.327 ...2020/IMG/scores/Log Likelihood_fixed.png}


Overfull \hbox (173.98659pt too wide) in paragraph at lines 327--328
 []\TU/SourceSerifPro(0)/m/n/10 Likelihood$[]\OML/cmm/m/it/10 ixed:pngLikelihoo
d[]ixed:bb[]$

LaTeX Warning: Citation 'kullback1951information' on page 9 undefined on input
line 336.

[9]

LaTeX Warning: Citation 'khemakhem2020variational' on page 10 undefined on inpu
t line 413.


Overfull \hbox (62.40047pt too wide) in paragraph at lines 413--414
\TU/SourceSerifPro(0)/m/n/10 that the hyperparameters used are the same as in t
he original iVAE paper [[]].
[10]))

LaTeX Warning: Empty bibliography on input line 10.

(../openreview/appendix.tex [11]) [12] [13] (./article.aux)

Package rerunfilecheck Warning: File `article.out' has changed.
(rerunfilecheck)                Rerun to get outlines right
(rerunfilecheck)                or use package `bookmark'.


LaTeX Warning: There were undefined references.


LaTeX Warning: Label(s) may have changed. Rerun to get cross-references right.


Package biblatex Warning: Please (re)run Biber on the file:
(biblatex)                article
(biblatex)                and rerun LaTeX afterwards.

 )
(see the transcript file for additional information)
Output written on article.pdf (13 pages).
Transcript written on article.log.
=== TeX engine is 'XeTeX'
Latexmk: References changed.
Latexmk: Non-existent bbl file 'article.bbl'
 No file article.bbl.
Latexmk: Missing input file: 'Reproducibility_Challenge_2020/IMG/scores/Mcc Score.png' from line
  'LaTeX Warning: File `Reproducibility_Challenge_2020/IMG/scores/Mcc Score.png' not found on input line 182.'
Latexmk: Missing input file: 'Reproducibility_Challenge_2020/IMG/scores/Log Likelihood.png' from line
  'LaTeX Warning: File `Reproducibility_Challenge_2020/IMG/scores/Log Likelihood.png' not found on input line 188.'
Latexmk: Missing input file: 'Reproducibility_Challenge_2020/IMG/scores/Mcc Score_fixed.png' from line
  'LaTeX Warning: File `Reproducibility_Challenge_2020/IMG/scores/Mcc Score_fixed.png' not found on input line 321.'
Latexmk: Missing input file: 'Reproducibility_Challenge_2020/IMG/scores/Log Likelihood_fixed.png' from line
  'LaTeX Warning: File `Reproducibility_Challenge_2020/IMG/scores/Log Likelihood_fixed.png' not found on input line 327.'
```

Can you check the path of the images? Also if possible, save the images with no space separated filenames: eg, `Log_Likelihood_fixed.png`

**Comment by GeertenRijsdijk** [comment]
I changed the image names, it seems to compile now. Let us know if there is anything else that is needed.

File: rijsdijk2021reproducing/journal/metadata.yaml [hunk 3]
```diff
@@ -5,59 +5,54 @@
...
 # You can get a DOI for your code from Zenodo, or an SWH identifier from
 # Software Heritage.
 #   see https://guides.github.com/activities/citable-code/
 code:
-  - url: https://github.com/rescience-c/template
-  - doi: 
-  - swh: 
+  - url: https://github.com/HiddeLekanne/Reproducibility-Challenge-iFlow
+  - doi: 10.5281/zenodo.4704869
+  - swh:
```
Comment (koustuvsinha) [inline rijsdijk2021reproducing/journal/metadata.yaml:55]: Please add the Software Heritage Foundation (SWH) uri, as described in readme

**Review by koustuvsinha — COMMENTED** [review]
Compilation works. Although, the submission is missing the SWH uri. Please check the readme to know how to get the SWH uri for your code folder.

**Comment by GeertenRijsdijk** [comment]
The SWH url has been added

**Comment by koustuvsinha** [comment]
Thanks for adding the SWH uri. There is only one thing left: please add a "[Re]" in the title. Otherwise LGTM.

@dodgejesse can you review the contents of the paper?

**Review by dodgejesse — APPROVED** [review]
Looks good to me. Authors could remove the page break on the second page (between the first-page summary and the introduction on page 3), but that's an aesthetic choice.

**Pull request merged**